torchvision/transforms/functional.py

```diff
@@ -124,7 +124,24 @@ def pil_to_tensor(pic):
     return img
 
 
-def convert_image_dtype(image: torch.Tensor, dtype: torch.dtype = torch.float) -> torch.Tensor:
+# torch.iinfo isn't scriptable so using this helper function
+# https://github.com/pytorch/pytorch/issues/41492
+def _max_value(dtype: int) -> int:
```

Comment:
it might be possible to memoize this if performance is an issue, or just convert to a lookup.